app/modules/dev/incident.py

```diff
@@ -21,15 +23,16 @@ def list_incidents(ack, logger, respond, client: WebClient, body):
         message = f"Is this an incident channel? {is_incident}\nIs dev channel? {is_dev_incident}"
         respond(message)
     logger.info("Listing incidents...")
-    incidents = incident_folder.list_incidents()
+    incidents = db_operations.list_incidents()
     respond(f"Found {len(incidents)} incidents")
     if len(incidents) > 10:
         incidents = incidents[:10]
     logger.info(json.dumps(incidents, indent=2))
     formatted_incidents = []
     for incident in incidents:
+        incident = {k: deserializer.deserialize(v) for k, v in incident.items()}
```

Comment:
Nice that you are using deserializer/serializer!

Reply:
Only testing for now, will update the logic of the db functions once I'm done refactoring the modules!